tests/testthat/test-explanation.R

```diff
@@ -74,9 +74,110 @@ test_that("Test functions in explanation.R", {
+  ex_list[[35]] <- explain(x_test, explainer, approach = c(rep("ctree", 3), rep("empirical", 1)), prediction_zero = p0, sample = TRUE)
+
+  # Ex 36: Explain different ctree mincriterion for different number of dependent variables, sample = TRUE
+  ex_list[[36]] <- explain(x_test, explainer, approach = "ctree", prediction_zero = p0, sample = TRUE, comb_indici = 2, comb_mincriterion = c(0.05, 0.95))
+
+  # Ex 37: Explain different ctree mincriterion for different number of dependent variables, sample = TRUE
+  ex_list[[37]] <- explain(x_test, explainer, approach = "ctree", prediction_zero = p0, sample = TRUE, comb_indici = 2, comb_mincriterion = c(0.95, 0.95))
+
+  # Ex 38: Explain different ctree mincriterion for different number of dependent variables, sample = TRUE
+  ex_list[[38]] <- explain(x_test, explainer, approach = "ctree", prediction_zero = p0, sample = TRUE, comb_indici = 0, comb_mincriterion = c(0.95, 0.95))
+
+  # Ex 39: Explain different ctree mincriterion for different number of dependent variables, sample = TRUE
+  ex_list[[39]] <- all((explain(x_test, explainer, approach = "ctree", prediction_zero = p0, sample = TRUE, comb_indici = 2, comb_mincriterion = c(0.95, 0.95)))$dt ==
+    (explain(x_test, explainer, approach = "ctree", prediction_zero = p0, sample = TRUE, mincriterion = 0.95))$dt)
+
+  # Ex 40: Explain different ctree mincriterion for different number of dependent variables, sample = TRUE
+  ex_list[[40]] <- all((explain(x_test, explainer, approach = "ctree", prediction_zero = p0, sample = TRUE, comb_indici = 0, comb_mincriterion = c(0.05, 0.95)))$dt ==
+                         (explain(x_test, explainer, approach = "ctree", prediction_zero = p0, sample = TRUE, mincriterion = rep(0.95, 4)))$dt)
+
+  # Ex 41: Explain different ctree mincriterion for different number of dependent variables, sample = TRUE
+  ex_list[[41]] <- all((explain(x_test, explainer, approach = "ctree", prediction_zero = p0, sample = TRUE, comb_indici = 0, comb_mincriterion = c(0.05, 0.95)))$dt ==
+                         (explain(x_test, explainer, approach = "ctree", prediction_zero = p0, sample = TRUE, mincriterion = 0.95))$dt)
+
+  # Ex 42: Explain different ctree mincriterion for different number of dependent variables, sample = TRUE
+  ex_list[[42]] <- all((explain(x_test, explainer, approach = "ctree", prediction_zero = p0, sample = FALSE, mincriterion = rep(0.95, 4)))$dt ==
+                         (explain(x_test, explainer, approach = "ctree", prediction_zero = p0, sample = FALSE, mincriterion = 0.95))$dt)
+
+  # Ex 43: Explain different ctree mincriterion for different number of dependent variables, sample = TRUE
+  ex_list[[43]] <- all((explain(x_test, explainer, approach = "ctree", prediction_zero = p0, sample = FALSE, mincriterion = c(rep(0.95, 2), rep(0.92, 2))))$dt ==
+                         (explain(x_test, explainer, approach = "ctree", prediction_zero = p0, sample = FALSE, mincriterion = 0.95))$dt)
```

Comment:
@aredelmeier These tests should be of the expect_equal-type (see below). With expect_known_value you are just checking that the results are the same as before, not whether the two components in all(...) actually are equal.
It would be great if you could fix that. No rush.